pkgs/applications/misc/todoist/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "todoist-${version}";
+  version = "0.13.1";
+
+  goPackagePath = "github.com/sachaos/todoist";
+
+  src = fetchFromGitHub {
+    owner = "sachaos";
+    repo = "todoist";
+    rev = "v${version}";
+    sha256 = "1kwvlsjr2a7wdhlwpxxpdh87wz8k9yjwl59vl2g7ya6m0rvhd3mc";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = {
+    homepage = https://github.com/sachaos/todoist;
+    description = "Todoist CLI Client";
+    license = lib.licenses.mit;
+    platforms = lib.platforms.unix;
+  };
```

Comment:
The package itself seems fine, but any reason you don't want to take maintainership for this? :)

Reply:
> any reason you don't want to take maintainership for this?

Yes, a few...

I've been using, developing features and adding packages to NixOS/nixpkgs for quite a few years already, so I've left my footprints in various places in the codebase and GitHub issues/PRs. This means I get mentioned frequently in GitHub and receive many NixOS-related emails, sometimes for packages or features I've not used or been interested in years, which just wastes my time. Therefore, I'd rather be selective about the things I'm interested in keep track of, so that I can better focus my time and so that I can contribute as much as I can possibly sustain, given that I do this on a voluntary basis.

In particular, for this package, this is a simple tool that I'm currently using (one of hundreds), so I'm not really interested in "maintaining" it, whether that includes updating the package, fixing build issues, or reviewing PRs for this package. In fact, this already made me waste more time than I wanted due to being a Go package that uses modules (something new in Go 1.11, apparently), which the nixpkgs manual, not being up-to-date regarding Go modules, actually sent me on the wrong path, until I figured out what was wrong. Note that I almost never use/develop in Go.

That said, I may update this package in the future (especially if the update fixes some annoying bug or the tool adds an interesting feature), I may fix build problems (if it affects me) and I may take a look and/or review PRs if someone mentions me. That said, I almost never do the latter, I'd rather prefer most PRs just get merged automatically and leave reviews/discussions for more complicated issues rather than simple package updates or build fixes / improvements...

Reply:
Thanks for elaborating!